remove_old_recordings.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+MAX_DAYS_AGO=14
+
+# remove folders including recordings older than MAX_DAYS_AGO
+# this should be run as a CronJob
+find /data/hiki/cctv/* -mindepth 1 -type d -ctime +$MAX_DAYS_AGO -exec rm -rf {} \;
```

Comment:
Would prefer this to be two finds, initially removing files matching a filename glob with -delete and subsequently removing the directories using -delete. It will fail to remove non empty ones but it is safer than the recursive rm approach.

Reply:
As I am not that good at BASHing (just a small joke!), that is not an easy task for me. Is it possible, to spare this file out from the pullrequest? just asking, because I finished implementing the path/filenamepattern already, waiting for a new pullrequest...